<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DIP VO</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Stereo Camera</string>
     </attribute>
     <widget class="QLabel" name="rawCamView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>240</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera1</string>
      </property>
     </widget>
     <widget class="QLabel" name="rectCamView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>640</width>
        <height>240</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Match View</string>
     </attribute>
     <widget class="QLabel" name="matchView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="text">
       <string>Match VIEW</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Marklands</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Feature Detector</string>
    </property>
    <widget class="QSlider" name="featureCountSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Feature Count</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>solvePNP</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="featureCountNumber">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sampleMKButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Sample Ref Marklands</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="poseMap">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="text">
     <string>pose map</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
